src/web-ui/src/public/components/Product.vue

```diff
@@ -1,18 +1,15 @@
 <template>
-  <div class="col-sm-12 col-md-4 col-lg-3 mb-4">
+  <div class="featured-product p-3 d-flex flex-column justify-content-between text-left">
     <!-- Card -->
     <router-link class="link" :to="{name:'ProductDetail', params: {id: product.id}, query: {feature: feature, exp: experimentCorrelationId}}">
-      <div class="card mx-auto h-100 product-card text-left">
+      <div>
         <img :src="productImageURL" class="card-img-top" :alt="product.name">
-        <div class="card-body">
-          <h5 class="card-title">{{ product.name }}</h5>
-          <p class="card-text">{{ product.description }}</p>
```

Comment:
I didn't realize that product description wasn't going to be used anymore. Is it used elsewhere in the app? Should it be removed from the data model completely?

Reply:
Descriptions are still used on the product detail page.

The goal with the product cards on this page is to have the same look as the product carousels used throughout the app. We can probably refactor to use the same component in the future.